client/python/gradio_client/client.py

```diff
@@ -929,20 +930,16 @@ def _upload(
                     output = [o for ix, o in enumerate(result) if indices[ix] == i]
                     res = [
                         {
-                            "is_file": True,
-                            "name": o,
+                            "path": o,
                             "orig_name": Path(f).name,
-                            "data": None,
                         }
                         for f, o in zip(fs, output)
                     ]
                 else:
                     o = next(o for ix, o in enumerate(result) if indices[ix] == i)
                     res = {
-                        "is_file": True,
-                        "name": o,
+                        "path": o,
                         "orig_name": Path(fs).name,
-                        "data": None,
                     }
                 uploaded.append(res)
```

Comment:
Do we need to check for the version here and do both? I don't think this is backwards compatible.

Reply:
It's ok - backwards compatibility is handled by another endpoint class entirely